src/flask_pyoidc/pyoidc_facade.py

```diff
@@ -261,6 +261,13 @@ def client_credentials_grant(self):
         On API call, token introspection will ensure that only valid token can
         be used to access your APIs.
 
+        Parameters
+        ----------
+        scope: list, optional
```

Comment:
Type should be `Optional[list]`.

Reply:
Numpy docstring format rules say to annotate docstring parameters like this only.

Reply:
Right, I've just reviewed those and the Google Python style guide and seems it's changed compared to the current format in this code base.
I'm thinking about dropping the types in docs and moving fully to proper type annotations.